test/unit/test_compiler_client.py

```diff
@@ -82,7 +79,7 @@ def test_compile_to_native_quil__returns_native_quil(
         protoquil=True,
     )
 
-    assert compiler_client.compile_to_native_quil(request) == CompileToNativeQuilResponse(
-        native_program="DECLARE ro BIT[1]\n",
-        metadata=None,
-    )
+    response = compiler_client.compile_to_native_quil(request)
+
+    assert response.native_program == "DECLARE ro BIT[1]\n"
+    assert response.metadata is not None
```

Comment:
I'm not totally sure why this test got `None` back before, but metadata being returned now doesn't seem like a bug, there is valid data in the response.